Dev

Mondzzz opened the pull request:
Current login page structure. Campus image and CSS are subject to change.

ItumelengMasuluke committed:
gitignore

ItumelengMasuluke committed:
initial

ItumelengMasuluke committed:
delete

ItumelengMasuluke committed:
Create .gitignore

ItumelengMasuluke committed:
init

ItumelengMasuluke committed:
Create Todo.txt

Mondzzz committed:
Merge pull request #1 from IMasuluke/Feature/addAxios

Added Axios to Feature/addAxios Branch

Mondzzz committed:
Added Axios to Feature/addAxios Branch

KekePhasha committed:
Footer/Component

Testing

KekePhasha committed:
Merge pull request #2 from IMasuluke/Features/add

Footer/Component

Mondzzz committed:
Feature addBootstrap

Mondzzz committed:
Merge branch 'dev' of https://github.com/IMasuluke/hackjam-frontend into dev

Mondzzz committed:
Merge pull request #4 from IMasuluke/Feature/addAxios

Feature/add axios

Mondzzz committed:
Added Bootstrap To Dependencies

Mondzzz committed:
Merge branch 'dev' into Feature/addAxios

Mondzzz committed:
Current Login Page

KekePhasha reviewed (approved):
Its seems good to me

KekePhasha committed:
Basic Css

KekePhasha committed:
StudentPortal Footer and Header

Mondzzz committed:
Merge pull request #6 from IMasuluke/StudentPortal_Design

Student portal design

Mondzzz committed:
Returning Student

KekePhasha committed:
Changes to Header

KekePhasha committed:
Changes to Footer

KekePhasha committed:
Delete Duplicate

KekePhasha committed:
Changing Structure and Adding Youtube file

Mondzzz committed:
Merge pull request #7 from IMasuluke/StudentHeader

Student header

Mondzzz committed:
Returning Student Changes

Mondzzz committed:
Merge pull request #8 from IMasuluke/Returning-Student

Returning student

KekePhasha committed:
StudentPortal

KekePhasha committed:
Merge pull request #9 from IMasuluke/StudentHeader

StudentPortal

Mondzzz committed:
Merge branch 'dev' into Returning-Student

KekePhasha committed:
Router

KekePhasha committed:
Merge branch 'dev' of https://github.com/IMasuluke/hackjam-frontend into dev

Mondzzz committed:
Student Portal URL added

KekePhasha committed:
News feed

KekePhasha committed:
Agenda

KekePhasha committed:
Basic Structure

KekePhasha committed:
Merge pull request #10 from IMasuluke/Structure

Structure

KekePhasha committed:
Adding More Stuff

KekePhasha committed:
Merge branch 'dev' into Structure

KekePhasha committed:
Merge pull request #11 from IMasuluke/Structure

Adding More Stuff

Mondzzz committed:
Merge branch 'dev' into Returning-Student

Mondzzz committed:
Merge pull request #12 from IMasuluke/Returning-Student

Latest Changes to Returning Student

Mondzzz committed:
Latest Changes to Returning Student

Mondzzz committed:
Merge branch 'dev' of https://github.com/IMasuluke/hackjam-frontend into dev

Mondzzz committed:
Additions to Returning Student Page

Mondzzz committed:
Merge pull request #14 from IMasuluke/Returning-Student

Returning student

Mondzzz committed:
Progress on Returning Student pages

KekePhasha committed:
Css Changes

KekePhasha committed:
New images

KekePhasha committed:
Events Page

KekePhasha committed:
Basic Changes

KekePhasha committed:
Merge branch 'dev' into Structure

KekePhasha committed:
Merge pull request #15 from IMasuluke/Structure

Structure

KekePhasha committed:
Events

KekePhasha committed:
Merge pull request #16 from IMasuluke/EventsPage

Events

Mondzzz committed:
Merge pull request #17 from IMasuluke/Returning-Student

Returning Student updates

Mondzzz committed:
Returning Student updates

Mondzzz committed:
Merge branch 'dev' of https://github.com/IMasuluke/hackjam-frontend into dev

Mondzzz committed:
Merge pull request #18 from IMasuluke/Returning-Student

Further additions to Returning Student

Mondzzz committed:
Further additions to Returning Student

KekePhasha committed:
Routing

KekePhasha committed:
Css Changes

KekePhasha committed:
Eventspage

KekePhasha committed:
New web pages

KekePhasha committed:
Small changes

KekePhasha committed:
Moving files

KekePhasha committed:
Merge pull request #19 from IMasuluke/EventsPage

Events page

Mondzzz committed:
Numerous changes to Returning-Student pages

State forms were replaced with dynamic tables.  Widths and heights in CSS need to be adjusted to fit preferred zoom magnification.

Mondzzz committed:
Merge branch 'dev' of https://github.com/IMasuluke/hackjam-frontend into dev

Mondzzz committed:
Numerous changes to Returning-Student pages

State forms have been replaced with dynamic tables. CSS widths and heights currently have to be adjusted to fit preferred zoom magnification. Will fix.

Mondzzz committed:
Merge branch 'dev' into Returning-Student

Mondzzz committed:
Merge pull request #21 from IMasuluke/Returning-Student

Returning student

Mondzzz committed:
Merge pull request #22 from IMasuluke/Returning-Student

Pedantic updates to App.js

Mondzzz committed:
Pedantic updates to App.js

Mondzzz committed:
Merge pull request #23 from IMasuluke/Returning-Student

Latest to Returning Student

Mondzzz committed:
Latest to Returning Student

Progress has been made on Log In page and Returning Student pages. Web pages are still being adjusted to fit 100% zoom magnification.

Mondzzz committed:
Merge pull request #24 from IMasuluke/Returning-Student

Comments for back end have been added.

Mondzzz committed:
Comments for back end have been added.

Please do not hesitate to seek any clarification on comments and code.

KekePhasha committed:
New page| Chats

KekePhasha committed:
New Page | Finance

KekePhasha committed:
Adding sticky component

KekePhasha committed:
Css changes

KekePhasha committed:
Adding Modals

KekePhasha committed:
Merge branch 'dev' into financePage

KekePhasha committed:
Merge pull request #25 from IMasuluke/financePage

Finance page

Mondzzz committed:
Merge pull request #26 from IMasuluke/Returning-Student

Progress on ReturningStudentWelcome Pages

Mondzzz committed:
Progress on ReturningStudentWelcomePages

KekePhasha committed:
Changes

KekePhasha committed:
Merge pull request #27 from IMasuluke/axios

Changes

Mondzzz committed:
Merge branch 'dev' into Returning-Student

Mondzzz committed:
Merge pull request #28 from IMasuluke/Returning-Student

Commitments

Mondzzz committed:
Commitments

KekePhasha committed:
changes

KekePhasha committed:
Merge pull request #29 from IMasuluke/axios

changes

Mondzzz committed:
Merge branch 'dev' into Returning-Student

Mondzzz committed:
Merge pull request #30 from IMasuluke/Returning-Student

Commitments

Mondzzz committed:
Commitments

Mondzzz committed:
Revert "Merge branch 'dev' into Returning-Student"

This reverts commit 2d11254c907a7c201b431740aa9a3819ffa3bf55, reversing
changes made to b73f063811015a38ec62c41a64038f3ef2084984.

Mondzzz committed:
Merge pull request #31 from IMasuluke/Returning-Student

Returning student

Mondzzz committed:
Merge

KekePhasha committed:
Need adjustments

KekePhasha committed:
Images

Mondzzz committed:
Merge pull request #32 from IMasuluke/Returning-Student

Minor changes

Mondzzz committed:
Minor changes
